operator/v1/types_network.go

```diff
@@ -428,7 +429,15 @@ type HybridOverlayConfig struct {
 	HybridOverlayVXLANPort *uint32 `json:"hybridOverlayVXLANPort,omitempty"`
 }
 
+// +kubebuilder:validation:XValidation:rule="self == oldSelf || self.has(mode)",message="ipsecConfig.mode is required"
```

Comment:
Sorry, I got this wrong, should be
```suggestion
// +kubebuilder:validation:XValidation:rule="self == oldSelf ||has(self.mode)",message="ipsecConfig.mode is required"
```

Reply:
Done